main.tex

```diff
@@ -1263,36 +1366,36 @@ \section{Conclusion}
 % Renamed and edited by Adrian PW
 
 The development of the \astropypkg package and cultivation of the \astropy
-ecosystem is still maintaining significant growth while improving stability,
+ecosystem are still maintaining significant growth while improving in stability,
 breadth, and reliability.
 As the \astropypkg core package becomes more mature, several subpackages have
 reached stability with a rich set of features that help astronomers worldwide to
-perform many daily tasks such as planning observations, analyzing data or
+perform many daily tasks, such as planning observations, analyzing data or
 simulation results, and writing publications.
-The strong emphasis that the \astropy project puts on reliability and code
-correctness helps users to trust the calculations performed with \astropypkg and
+The strong emphasis that the \astropy project puts on reliability and high coding
+standards helps users to trust the calculations performed with \astropypkg and
 to publish reproducible results.
-At the same time, the \astropy ecosystem and core package are growing: new
-functionality is still being contributed, and new affiliated packages are being
+At the same time, the \astropy ecosystem and core package are both growing: new
+functionality is still being contributed and new affiliated packages are being
 developed to support more specialized needs.
 
 The \astropy project is also spreading awareness of best practices in
 community-driven software development.
 This is important as most practicing astronomers were not explicitly taught
 computer science and software development, despite the fact that a major
-fraction of nearly every astronomer's workload today is related to software use
+fraction of many astronomers' workload today is related to software use
 and development.
 The \astropypkg package leads by example, showing all interested astronomers how
-modern tools like \texttt{git} version control or continuous integration can
+modern tools like \texttt{git} version control or CI testing can
 increase the quality, accessibility, and discoverability of astronomical
 software without overly complicating the development cycle.
 Within \astropy, all submitted code is reviewed by at least one, but typically
 more, members of the \astropy community, who provide feedback to contributors
-to help improve their skills.
+to uphold coding best practices.
```

Comment:
Personally, I don't provide feedback just for the sake of others improving skills. My first priority is the code going into the package, people improving skills is a bonus, so I reworded this.

Reply:
I do think we want to point out that this helps the contributors improve as developers, but don't feel strongly.

Reply:
How about "to uphold and disseminate best coding practices"?

Reply:
how about "which helps to improve their skills"? My point here was (I wrote the section originally) that working on astropy help the next generation of astronomers become better coding; this is intended to tie in with the sentences before that say "astronomers are not taught computer science". In that sense, astropy is an educational institution and students contributing can tell their advisors "I contribute because I learn something there".
And, whatever your intend is in providing feedback, the result is that that contributors learn something.